src/mjolnir/util.cc

```diff
@@ -25,6 +25,17 @@
 namespace valhalla {
 namespace mjolnir {
 
+// Temporary files used during tile building
+const std::string ways_file = "ways.bin";
```

Comment:
could we move stuff that we dont want to expose via the header into an anonymous namespace? specifically i mean this for the functions below as well

Reply:
moved the string to an anonymous namespace and the remove method to be a lambda. Other functions in this file are exposed via header and used elsewhere in mjolnir